GetandTake/appsettings.json

```
@@ -5,5 +5,9 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=EPTRANKW0038\\SQLEXPRESS;Database=Northwind;Trusted_Connection=True;"
+  },
+  "MaximumAmountOfProduct" : 12
```

After creating a separate C# class with config description, you can easily transform your value into section like this:
```suggestion
  "products" : {
        "maximumAmount": 12
  }
```

In this way you will have separate C# class that matches `products` configuration and its properties will contain info read from JSON